src/components/Login.tsx

```diff
@@ -60,7 +61,7 @@ export default function Login() {
             </div>
             <div className="text-sm">
                 <a
-                  href="/register"
+                  href="#"
```

Comment:
can import Link from react-router-dom and use: 
<Link to="/register"
className="font-medium text-indigo-600 hover:text-indigo-500" > 
</Link>

Its more common i think